app/assets/javascripts/provider/admin/apiconfig/services/integration.js.coffee

```diff
@@ -83,6 +87,10 @@ $(document).on 'initialize', '#proxy', ->
       """
       $(".feedback").addClass("no-test")
       $("#client-request").hide()
+
+  $(document).ready ->
```

Comment:
Hmmm. This has implications as well to the new Configuration page (the one introduced with APIAP), which should not depend on javascript to show the test curl command.

Reply:
🤔 Ok, then I will try to do it purely without JS but behaving the same way instead of rendering something empty